Simplenote/src/main/java/com/automattic/simplenote/Simplenote.java

```diff
@@ -212,6 +218,30 @@ public boolean isInBackground() {
         return mIsInBackground;
     }
 
+    public void login(String email, String spToken) {
```

Comment:
thanks for adding this function! I could see it being a little confusing having a login function taking two strings that aren't username-and-password, but I think we can evaluate that in a follow-up issue if it would hold up this pr.

Reply:
I renamed the function to `loginWithToken` to make it clearer.